hieradata/common.yaml

```diff
@@ -337,6 +337,15 @@ govuk::apps::govuk_crawler_worker::root_urls:
   - 'https://assets.publishing.service.gov.uk/'
   - 'https://www.gov.uk/'
 
+govuk::apps::govuk_delivery::redis_host: 'redis-1.backend'
```

Comment:
Should probably use `"%{hiera('sidekiq_host')}"` as elsewhere in this file. I know this is for configuring Celery rather than Sidekiq, but I can't think of a reason to separate them.